### PR TITLE
payload reword

### PR DESCRIPTION
**Pull request by lillie-dae**
### Description

Fixes # .

A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Types of changes

- [ ] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] All tests passed locally.
- [ ] [Documentation comments](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments) included/updated.
- [ ] User guide updated.
- [ ] I have updated the changelog

### CODE REVIEW

File: src/InformaticsGateway/Services/Http/PayloadsController.cs [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    public interface IPaginatedApi<T>
+    {
+        Task<long> CountAsync();
+
+        Task<IList<T>> GetAllAsync(int? skip, int? limit);
+    }
+
+    class PayloadService : IPayloadService
+    {
+        private readonly IPayloadRepository _payloadRepository;
```
Comment (lillie-dae) [inline src/InformaticsGateway/Services/Http/PayloadsController.cs:102]: TODO move these to own files appropriate foldered

File: src/InformaticsGateway/Services/DicomWeb/IStreamsWriter.cs [hunk 2]
```diff
@@ -251,12 +251,12 @@ private void AddFailure(DicomStatus dicomStatus, StudySerieSopUids uids = defaul
                 _resultDicomDataset.Add(failedSopSequence);
             }
 
-            if (uids is not null)
+            if (aids is not null)
```
Comment (lillie-dae) [inline src/InformaticsGateway/Services/DicomWeb/IStreamsWriter.cs:254]: fix typo

File: docker-compose/docker-compose.yml [hunk 3]
```diff
@@ -21,8 +21,8 @@ services:
       - 5672:5672
       - 15672:15672
     environment:
-      RABBITMQ_DEFAULT_USER: "rabbitmq"
```
Comment (neildsouth) [inline docker-compose/docker-compose.yml:24]: do we need to change this ?

File: .editorconfig [hunk 4]
```diff
@@ -98,7 +98,7 @@ dotnet_naming_style.i_prefix_pascal_case.required_prefix    = I
 dotnet_naming_style.i_prefix_pascal_case.capitalization     = pascal_case
 
 # name all constant fields using PascalCase
-dotnet_naming_rule.constant_fields_should_be_pascal_case.severity   = error
+dotnet_naming_rule.constant_fields_should_be_pascal_case.severity   = suggestion
```
Comment (mocsharp) [inline .editorconfig:101]: We want to keep it as an `error` so all is consistent.

File: src/Api/Storage/PatientDetails.cs [hunk 5]
```diff
@@ -0,0 +1,74 @@
+using System;
```
Comment (mocsharp) [inline src/Api/Storage/PatientDetails.cs:1]: Would you like to discuss the change request?

File: src/InformaticsGateway/Common/StudySeriesSopAids.cs [hunk 6]
```diff
@@ -16,7 +16,7 @@
 
 namespace Monai.Deploy.InformaticsGateway.Common
 {
-    public record StudySerieSopUids
+    public record StudySeriesSopAids
```
Comment (mocsharp) [inline src/InformaticsGateway/Common/StudySeriesSopAids.cs:19]: Sorry, what does `Aids` stand for in this case?

File: src/InformaticsGateway/Services/Http/PayloadsController.cs [hunk 7]
```diff
@@ -0,0 +1,119 @@
+using System;
```
Comment (mocsharp) [inline src/InformaticsGateway/Services/Http/PayloadsController.cs:1]: Please update user guide with all APIs. Thanks

**Pull request closed without merging**